include/libdnf5-cli/output/transactionlist.hpp

```diff
@@ -27,7 +27,9 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 
 namespace libdnf5::cli::output {
 
-LIBDNF_CLI_API void print_transaction_list(std::vector<libdnf5::transaction::Transaction> & ts_list);
+LIBDNF_CLI_API void print_transaction_list(
```

Comment:
I think this will break ABI compatibility. Could we possibly load the items inside this function? So like moving `history.get_transaction_item_counts(transactions);` inside this without modifying the API?